app/google-services.json

```diff
@@ -1,27 +1,27 @@
 {
   "project_info": {
-    "project_number": "528542541788",
-    "firebase_url": "https://veranda-1d914.firebaseio.com",
-    "project_id": "veranda-1d914",
-    "storage_bucket": "veranda-1d914.appspot.com"
+    "project_number": "855027673764",
+    "firebase_url": "https://dynamiclinks-test-35c22.firebaseio.com",
+    "project_id": "dynamiclinks-test-35c22",
+    "storage_bucket": "dynamiclinks-test-35c22.appspot.com"
```

Comment:
We will want to revert this back to our google-services.json file.  Our dynamic link hostname is viasatconnect.page.link